frame/staking/src/lib.rs

```diff
@@ -1149,6 +1148,8 @@ decl_error! {
 		TooManyTargets,
 		/// A nomination target was supplied that was blocked or otherwise not a validator.
 		BadTarget,
+		/// Too cannot add another lock to the account.
+		TooManyLocks,
```

Comment:
It does look a bit odd to me that we're repeating the same error variant on a different pallet, can do a follow up for DRYing this up.

Reply:
i just dont see a more appropriate place to put it. Not general enough to make it into DispatchResult and stuff